content/grants/2023/tinygrad.md

```diff
@@ -0,0 +1,24 @@
++++
+
+title = "Tinygrad ML Library in Hoon"
+date = "2023-03-02"
+
+[taxonomies]
+grant_type = ["Request"]
+grant_category = ["Core Dev"]
+
+[extra]
+image = ""
+description = "Implement a small machine learning library (and jets!) in hoon."
+reward = "TBD"
+assignee = [""]
+champion = [""]
+grant_id = "P1003"
+completed = false
+canceled = false
+
++++
+
+Geohot has published the [Tinygrad library](https://github.com/geohot/tinygrad) for doing basic machine learning. This library is small and simple which makes it a potentially good target for jetting. Doing so would give basic ML functionality in Urbit.
+
+Implement Tinygrad (with jets) in Urbit
```

Comment:
I would note that we should standardize on the representation of ndarray data in Urbit first.